app/va/identifier.py

```diff
@@ -40,6 +40,8 @@ class UnsupportedIdentifierException(Exception):
 
 
 def transform_to_fhir_format(recipient_identifier):
+
+    identifier_type = "Unknown Type"
```

Comment:
```suggestion
```

Reply:
Line 51 below needs to change `identifier_type` to `recipient_identifier.id_type`